README.md

```diff
@@ -2114,6 +2114,9 @@ describe('Admin View', function() {
   assert.equal(json.name, 'Daniel-san');
 ```
 
+### Releasing new versions
+1. ?
```

Comment:
@danielspaniel any chance you could help me document the release process? A comment here with the steps you use is good enough, and I'll update the content.

Reply:
you do something like 

 npm version patch  or npm version minor  etc .. 
 npm publish 
 git push --tags

 I wonder why we are doing all the hokus pokus of changing the ownership. your a maintainer and you can do what you want . what is difference who owned repo. seems like waste of time

Reply:
Great, thanks. I still think it's a worthwhile exercise. In my opinion moving it to the org increases the bus factor because I mightn't always be available to help out.